### ISSUE TITLE
Semi-supervised GAN for Classification of Multispectral Imagery Acquired by UAVs

### ISSUE THREAD

**Issue by llien30** [mn1204/paper_summary#20]
# INFO

## Author
Hamideh Kerdegari, Manzoor Razaak, Vasileios Argyriou, Paolo Remagnino
## Conference or Year
24 May 2019
## Link
- [arXiv](https://arxiv.org/abs/1905.10920)
- [official GitHub]()

# Abstract

# Proposed Method
we apply **"Semi-supervised GAN"** to the agricultural task.
## Typical GAN
Discriminator is only used as a binary classifier(classification classes = real / fake)

## semi-supervised GAN
Discriminator is used as a n+1 class classifier( classification classes = n classes / fake )
<img width="644" alt="Screen Shot 2020-02-27 at 10 24 49" src="https://user-images.githubusercontent.com/51707667/75403545-64fddf00-594b-11ea-823a-3a85ecb424a7.png">

- use DCGAN network
- Sigmoid activation function → Softmax (to enable classification)
- Library : Keras(tf background)
- optimizer : Adam with momentum (lr=0.0002(G), 0.5(D))
- batch size : 32 (batch normalization is used for both G and D)
- Generator : use ReLU activation(except last layer using Tanh)
- Discriminator : use LeakyReLU activation
- latent vector dim : 100
- the number of the feature maps : 256, 128, 64, 32, 1
<img width="787" alt="Screen Shot 2020-02-27 at 10 24 27" src="https://user-images.githubusercontent.com/51707667/75403521-56afc300-594b-11ea-82df-c60fab4a87b5.png">

# Evaluation

# Contribution
proposed Semi-supervised classification GAN model 
→ success to classify multi-spectral images

# Discussion, Future Work

## Date
2020/02/27
